runtime/shared_common/CacheMap.cpp

```diff
@@ -4558,14 +4585,18 @@ SH_CacheMap::printAllCacheStats(J9VMThread* currentThread, UDATA showFlags, SH_C
 							CACHEMAP_PRINT((J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE), J9NLS_SHRC_CM_PRINTSTATS_STALE);
 						}
 					}
-				} else if (J9SHR_DATA_TYPE_UNUSED1 == type) {
+				} else if (J9SHR_DATA_TYPE_STARTUP_HINTS == type) {
 					if ((PRINTSTATS_SHOW_BYTEDATA == (showFlags & PRINTSTATS_SHOW_BYTEDATA))
 						|| (isStale && showAllStaleFlag)
 					) {
-						CACHEMAP_PRINT6((J9NLS_DO_NOT_PRINT_MESSAGE_TAG), J9NLS_SHRC_CM_PRINTSTATS_UNUSED1_DISPLAY, ITEMJVMID(it), (UDATA)it, J9UTF8_LENGTH(pointer), J9UTF8_DATA(pointer), BDWDATA(bdw), BDWLEN(bdw));
+						J9SharedStartupHintsDataDescriptor* hints = (J9SharedStartupHintsDataDescriptor*)BDWDATA(bdw);
+						CACHEMAP_PRINT6((J9NLS_DO_NOT_PRINT_MESSAGE_TAG), J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS_DISPLAY, ITEMJVMID(it), (UDATA)it, J9UTF8_LENGTH(pointer), J9UTF8_DATA(pointer), BDWDATA(bdw), BDWLEN(bdw));
+						CACHEMAP_PRINT12((J9NLS_DO_NOT_PRINT_MESSAGE_TAG), J9NLS_SHRC_CM_PRINTSTATS_STARTUP_HINTS_DISPLAY_DETAIL, hints->flags, hints->heapSize1, hints->heapSize2, hints->unused1, hints->unused2, hints->unused3, hints->unused4, hints->unused5,
```

Comment:
I'm ok with not printing any of the unused data. When the data starts to be used then it can be printed as part of that change. It means an older VM won't display data created by a newer VM, but I don't think there is any requirement to do so.